packages/talker_logger/lib/src/settings/talker_logger_settings.dart

```diff
@@ -40,4 +41,7 @@ class TalkerLoggerSettings {
 
   /// Field for enable and disable colored logs
   final bool enableColors;
+
+  /// Flag for default order of logs to be latest first (at the top). Default false
+  final bool displayLatestFirst;
```

Comment:
It is not a TalkerLoggerSettings field.
This field should be implemented in TalkerFlutter controller only

Reply:
I moved the logic into TalkerFlutterController, thanks @Frezyx 

However if we don't have a flag in TalkerLoggerSettings, then we won't be able to have a default sorting order, it will always be earliest first.